rosidl_generator_dotnet/resource/msg.cs.em

```diff
@@ -72,7 +88,12 @@ public class @(type_name) : IMessage {
             (NativeWriteField@(get_field_name(type_name, member.name))Type)Marshal.GetDelegateForFunctionPointer(
             native_write_field_@(member.name)_ptr, typeof(NativeWriteField@(get_field_name(type_name, member.name))Type));
 @[    else]@
-// TODO: Nested types are not supported
+        IntPtr native_get_field_@(member.name)_HANDLE_ptr =
+            dllLoadUtils.GetProcAddress(nativelibrary, "@(msg_typename)__native_get_field_@(member.name)_HANDLE");
```

Comment:
For naming consistency with read_field/write_field above:

```suggestion
            dllLoadUtils.GetProcAddress(nativelibrary, "@(msg_typename)__get_field_@(member.name)_HANDLE");
```

Reply:
ok